content/legal.rst

```diff
@@ -22,121 +22,74 @@ Licenses
-    :alt: Download Odoo Enterprise Agreement (FR)
-    :target: https://www.odoo.com/documentation/12.0/odoo_enterprise_agreement_fr.pdf
-
-.. |view_enterprise_fr| image:: legal/img/txt.svg
-    :alt: View Odoo Enterprise Agreement (FR)
-    :target: legal/terms/i18n/enterprise_fr.html
-
-.. |download_partnership_fr| image:: legal/img/pdf.svg
-    :alt: Download Odoo Partnership Agreement (FR)
-    :target: https://www.odoo.com/documentation/12.0/odoo_partnership_agreement_fr.pdf
-
-.. |view_enterprise_nl| image:: legal/img/txt.svg
-    :alt: View Odoo Enterprise Agreement (NL)
-    :target: legal/terms/i18n/enterprise_nl.html
-
-.. |view_partnership_fr| image:: legal/img/txt.svg
-    :alt: View Odoo Partnership Agreement (FR)
-    :target: legal/terms/i18n/partnership_fr.html
-
-.. |download_terms_of_sale_fr| image:: legal/img/pdf.svg
-    :alt: Download Odoo Terms of Sale (FR)
-    :target: https://www.odoo.com/documentation/12.0/terms_of_sale_fr.pdf
-
-.. |view_terms_of_sale_fr| image:: legal/img/txt.svg
-    :alt: View Terms of Sale (FR)
-    :target: legal/terms/i18n/terms_of_sale_fr.html
-
-.. |download_enterprise_nl| image:: legal/img/pdf.svg
-    :alt: Download Odoo Enterprise Agreement (NL)
-    :target: https://www.odoo.com/documentation/12.0/odoo_enterprise_agreement_nl.pdf
-
-.. |download_partnership_nl| image:: legal/img/pdf.svg
-    :alt: Download Odoo Partnership Agreement (NL)
-    :target: odoo_partnership_agreement_nl.pdf
-
-.. |download_enterprise_de| image:: legal/img/pdf.svg
-    :alt: Download Odoo Enterprise Agreement (DE)
-    :target: https://www.odoo.com/documentation/12.0/odoo_enterprise_agreement_de.pdf
-
-.. |view_enterprise_de| image:: legal/img/txt.svg
-    :alt: View Odoo Enterprise Agreement (DE)
-    :target: legal/terms/i18n/enterprise_de.html
-
-.. |download_partnership_de| image:: legal/img/pdf.svg
-    :alt: Download Odoo Partnership Agreement (DE)
-    :target: https://www.odoo.com/documentation/12.0/odoo_partnership_agreement_de.pdf
-
-.. |view_partnership_de| image:: legal/img/txt.svg
-    :alt: View Odoo Partnership Agreement (DE)
-    :target: legal/terms/i18n/partnership_de.html
-
-.. |download_enterprise_es| image:: legal/img/pdf.svg
-    :alt: Download Odoo Enterprise Agreement (ES)
-    :target: https://www.odoo.com/documentation/12.0/odoo_enterprise_agreement_es.pdf
-
-.. |view_enterprise_es| image:: legal/img/txt.svg
-    :alt: View Odoo Partnership Agreement (ES)
-    :target: legal/terms/i18n/enterprise_es.html
-
-.. |download_partnership_es| image:: legal/img/pdf.svg
-    :alt: Download Odoo Partnership Agreement (ES)
-    :target: https://www.odoo.com/documentation/12.0/odoo_partnership_agreement_es.pdf
-
-.. |view_partnership_es| image:: legal/img/txt.svg
-    :alt: View Odoo Partnership Agreement (ES)
-    :target: legal/terms/i18n/partnership_es.html
+.. list-table::
+   :header-rows: 1
+
+   * -
+     - English
+     - Français
+     - Nederlands
+     - Deutsch
+     - Español
```

Comment:
the headers are aligned differently than the rows inside the table, which sometimes makes it difficult to find why language the files correspond to (in a wide browser window)

Seems like we use custom CSS[^1] for that, so it's easy to fix:
- either we remove the custom CSS and make everything left-aligned
- or we center all rows, and make `text-align: center` also apply to the TDs


[^1]: https://github.com/odoo/documentation/blob/0d474685b551f3139a1efe4d6ab096fb523a41d3/static/css/legal.css#L7-L9

Reply:
Less CSS is better CSS.